app/src/main/res/values/styles.xml

```diff
@@ -209,6 +209,8 @@
   <style name="PledgeButton">
     <item name="android:layout_gravity">bottom</item>
     <item name="android:textAllCaps">false</item>
+    <item name="android:textColor">@color/white</item>
+    <item name="android:backgroundTint">@drawable/ksr_button_state</item>
```

Comment:
This a color. Since it's not used for all of our buttons it should be renamed to `button_reward_pledge`.